docs/docs/build/dashboards/canvas-components/misc.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Misc Components"
+sidebar_label: "Misc"
```

Comment:
Nit: The shorthand "Misc" feels a little too informal. "Miscellaneous" may be better